gradle/javadoc.gradle

```diff
@@ -90,5 +91,18 @@ allprojects {
                     |
                     | </script>'''.stripMargin()
         }
+
+        afterEvaluate {
+            destinationDir = new File(rootDir, "build/docs/${project.archivesBaseName}")
+            def version = project.version as String
+            if (project.name != 'bson') {
+                options.linksOffline(*buildDocsUrl(version, 'bson'))
```

Comment:
How come offline links and not regular links?

Reply:
This is a chicken and egg scenario, because we don't have them published anywhere, we can't read the url and then link off to them.  See: https://docs.oracle.com/javase/7/docs/technotes/tools/windows/javadoc.html#linkoffline